syntax = "proto3";
package pwservice;

service PasswordService {
  rpc ChangePassword (ChangePasswordRequest) returns (PamReply) {}
}

message ChangePasswordRequest {
  string user = 1;
  string old_password = 2;
  string new_password = 3;
}

message PamReply {
  bool success = 1;
  string message = 2;
}
